nixos/modules/services/security/bitwarden-directory-connector.nix

```diff
@@ -0,0 +1,327 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.services.bitwarden_directory_connector;
+
+  ldap_data = builtins.toJSON cfg.ldap;
+  sync_data = builtins.toJSON cfg.sync;
+in {
+  imports = [];
+
+  options.services.bitwarden_directory_connector = {
```

Comment:
Underscores in nixos option names are very rare. camelCase is the norm, and dashes are sometimes used, particularly in the names of programs and services, but underscores don't fit in. I'd use dashes here, I think.

Reply:
done